server/api.js

```diff
@@ -816,4 +817,42 @@ router.post("/logout", async (req, res) => {
 	}
 });
 
+router.get("/bid-detail/:id", async (req, res) => {
```

Comment:
This is more standard for rest endpoints:

```suggestion
router.get("/bid/:id", async (req, res) => {
```